### PR TITLE
Vinit work

### PR DESCRIPTION
**Pull request by vdongre304**

### CODE REVIEW

**Review by Jestinxavier — APPROVED** [review]

**Review by yogesh545 — APPROVED** [review]

File: vinitD/html/asgnmnt1/assign1.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=<h1>, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline vinitD/html/asgnmnt1/assign1.html]: change title title should be related to program

File: vinitD/html/asgnmnt1/assign2.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=<h1>, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline vinitD/html/asgnmnt1/assign2.html]: change title title should be related to program

File: vinitD/html/asgnmnt1/assign3.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=<h1>, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline vinitD/html/asgnmnt1/assign3.html]: change title title should be related to program

File: vinitD/html/asgnmnt1/assign4.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (KomalShinde813) [inline vinitD/html/asgnmnt1/assign4.html]: change title title should be related to program

File: vinitD/html/asgnmnt4/Table2.html [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+        padding: 10px;
+        margin: 10px;
+      }
+    </style>
+  </head>
+  <body>
+    <table>
+      <caption>
+        <strong style="color: rgb(131, 52, 204); font-size: 30px"
+          >Jan's spring term afternoon schedule</strong
```
Comment (KomalShinde813) [inline vinitD/html/asgnmnt4/Table2.html]: > tag should close at above line and </strong> should at below line

File: vinitD/html/asgnmnt4/Table2.html [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+        <td></td>
+        <td rowspan="2">office</td>
+        <td rowspan="8">office hours by appt</td>
+        <td rowspan="2">office</td>
+      </tr>
+      <tr>
+        <th>1:30</th>
+        <td rowspan="2">Dept meeting</td>
+      </tr>
+
```
Comment (KomalShinde813) [inline vinitD/html/asgnmnt4/Table2.html]: remove this extra line

**Review by KomalShinde813 — CHANGES_REQUESTED** [review]

**Review by KomalShinde813 — APPROVED** [review]

**Pull request merged**